README.md

```diff
@@ -109,6 +110,8 @@ Commands:
   dkb       dkb.de with chipTAN QR (postbox)
   elster    elster.de with path to .pfx certfile as username (postbox)
   ing       banking.ing.de with photoTAN (postbox)
+  handyvertrag
+            handyvertrag.de (invoices)
```

Comment:
here we should also mention "(invoices, call logs)". I will add this to the other plugins as well.

Reply:
I've found "itemised bill", "itemised call record" and "itemisation report" for "Einzelverbindungsnachweis". "call log" translates back to "Anrufliste". Certainly anything is better than "connection report" :)

Reply:
i think "call record" would be something, anyone would associate with "EVN". I'll change it after merging.